### PR TITLE
[FIX] snailmail: Fix base snailmail layouts (15.0)

### PR DESCRIPTION
**Pull request by delvsola**
This PR fixes the basic layouts for snailmail in odoo 15.0 due to changes having been made to the invoices themselves and the service provider changing their validation area in the future.

This PR is not to be forwarded and is exclusive to 15.0, saas-15.2 and saas-15.3 as other versions require different tweaks to their layouts.

### CODE REVIEW

File: addons/snailmail/static/src/scss/snailmail_external_layout_asset.scss [hunk 1]
```diff
@@ -1,58 +1,49 @@
 /*Modifications for the Standard and Boxed document layouts */
-.header.o_company_1_layout .o_boxed_header {
-	max-height: 40mm;
-}
-
 .article.o_report_layout_standard.o_company_1_layout, .article.o_report_layout_boxed.o_company_1_layout {
     > .pt-5 {
+        padding-top: 0 !important;
```
Comment (fda-odoo) [inline addons/snailmail/static/src/scss/snailmail_external_layout_asset.scss:4]: tab issues :)

**Review by fda-odoo — CHANGES_REQUESTED** [review]

File: addons/snailmail/static/src/scss/snailmail_external_layout_asset.scss (hunk 1 above)
Comment (delvsola) [reply addons/snailmail/static/src/scss/snailmail_external_layout_asset.scss:4]: Nice catch, fixed !

**Comment by fda-odoo** [comment]
@robodoo r+

**Comment by delvsola** [comment]
@robodoo retry

**Comment by delvsola** [comment]
@robodoo retry

**Comment by d-fence** [comment]
robodoo retry

**Comment by fda-odoo** [comment]
@robodoo r+

**Pull request closed without merging**